01-make_graph.py

```diff
@@ -83,6 +83,8 @@ def source_location(meta_yaml):
         elif r.status_code != 200:
             print('Something odd happened to this recipe '
                                '{}'.format(r.status_code))
+            with open('bad.txt', 'a') as f:
+                f.write('{}\n'.format(name))
```

Comment:
Might want to also put in what the error code is

Reply:
maybe, but we directly parse bad without a second look I think